vcd/resource_vcd_external_network.go

```diff
@@ -0,0 +1,381 @@
+package vcd
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdExternalNetwork() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVcdExternalNetworkCreate,
+		Delete: resourceVcdExternalNetworkDelete,
+		Read:   resourceVcdExternalNetworkRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"ip_scope": &schema.Schema{
+				Type:        schema.TypeList,
+				Required:    true,
+				ForceNew:    true,
+				Description: "A list of IP scopes for the network",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"is_inherited": &schema.Schema{
+							Type:        schema.TypeBool,
+							Required:    true,
```

Comment:
Correct me if I'm wrong, but we don't even have this field in the GUI and it's not used often, hence suggestion is to make it optional and default to false.

Reply:
Yes, we don't have in UI, yet by API it is mandatory.

Reply:
Checked against created in UI. Changed to optional and false.